diff --git a/CMakeLists.txt b/CMakeLists.txt
index fad0497f5..b8d382193 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -321,7 +321,7 @@ endif()
 # necessary to include this for the generated "config.h" header
 include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
-add_library(${LIBNAME} SHARED
+add_library(${LIBNAME} STATIC #SHARED
     src/libliquid.c
     $<TARGET_OBJECTS:agc>
     $<TARGET_OBJECTS:audio>
